addons/web/static/src/legacy/js/core/utils.js

```diff
@@ -240,9 +241,32 @@ class UnknownPatchError extends Error {
     }
 }
 
+// not sure how to tell eslint that _Markup is defined in
+// addons/web/static/src/js/libs/underscore.js, though TBF it's not a *great*
+// solution, it's necessary for Markup to be available to QWebJS.
```

Comment:
Since it's declared globally, you should probably add it to the list of available globals in `test_eslint.py` (and in your local eslint config) or use an eslint global comment to declare it just in this block. Although in this case I'm confused as to why we want to add this to a library's code rather than having it inside its own module.

Reply:
It was not added to a library's core, somewhat confusingly `addons/web/static/src/js/libs/underscore.js` (now `addons/web/static/src/legacy/js/libs/underscore.js`) is *extensions* to underscore, not the actual underscore.

But I've moved everything back to this module, because there are situations where underscore.js gets loaded twice, and the workarounds in order to avoid the `class` triggering a parse error the second time around were crappy.